### PR TITLE
fix: allow special char, digits in search input of service overview

### PR DESCRIPTION
**Pull request by shubhamv-ss**
## Description
- Allow special chars, and digits in search services

## Why
- The regular expression that previously restricted search input to text-only has been removed. 
- Also, there is no validation for **service names** during the creation process. 
- This modification establishes consistency between the **service name** field in the **Create Service Card** and the **search services** functionality.
- 

https://github.com/user-attachments/assets/b622bb30-9221-4647-a106-e59f8cf55cf2

## Changelog Entry

```
- **Service Overview**
  - allowed special chars and digits in search services field
```
## Issue

- https://github.com/eclipse-tractusx/portal-frontend/issues/1241

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### BugFixes
+
+- **Service Overview**
+  - search services field should allow speacial chars, digits
```
Comment (evegufy) [inline CHANGELOG.md:8]: Please put changelog update in PR descriptions to avoid conflicts

@manojava-gk @lavanya-bmw @kunalgaurav-bmw Please review

**Review by evegufy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shubhamv-ss) [reply CHANGELOG.md:8]: Updated!

**Comment by evegufy** [comment]
@manojava-gk @lavanya-bmw @kunalgaurav-bmw Please review

**Review by manojava-gk — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lavanya-bmw) [reply CHANGELOG.md]: > Updated!

Please remove CHANGELOG.md changes as @evegufy requested

Comment (shubhamv-ss) [reply CHANGELOG.md]: Done, Please check @lavanya-bmw

**Comment by shubhamv-ss** [comment]
Hi @evegufy @manojava-gk @lavanya-bmw ,
Can any one of you check this ticket and merge if possible?

Thanks!

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Comment by shubhamv-ss** [comment]
Hi @evegufy ,
Please check if it's ready to be merged.

**Review by evegufy — APPROVED** [review]

**Pull request merged**